pkg/falco/helpers.go

```diff
@@ -55,6 +56,16 @@ func (p *PluginConfigInfo) initConfigString() string {
 // (i.e. falco.yaml) loading one or more plugins.
 func NewPluginConfig(configName string, plugins ...*PluginConfigInfo) (run.FileAccessor, error) {
 	var buf bytes.Buffer
+
+	// If we are running a newer Falco version with
+	// the container plugin, enforce it to the
+	if _, err := os.Stat(FalcoContainerPluginLibrary); err == nil {
+		plugins = append(plugins, &PluginConfigInfo{
```

Comment:
If the container plugin is present, append it when generating the plugin config.